File: code.py
import random
import time
random.seed(time.time())
print("-----RANDOM WHEEL OF FORTUNE-----")
n = int(input("Hi, enter the number of elements you want\n"))
while n <= 0:
    print("Error: number below or equal to 0. Try again")
    n = int(input())
print("What type of elements do you want? (i/d/s/c)")
t = input().lower() 
elements = []
if t == 'i':
    for i in range(n):
        val = int(input(f"Enter the element {i + 1}\n"))
        elements.append(val)
elif t == 'd':
    for i in range(n):
        val = float(input(f"Enter the element {i + 1}\n"))
        elements.append(val)
elif t == 's' or t == 'c':
    for i in range(n):
        val = input(f"Enter the element {i + 1}\n")
        elements.append(val)
if elements:
    random_index = random.randint(0, n - 1)
    print(f"The random element is {elements[random_index]}")

File: code.cpp
#include <iostream>
#include <cstdlib>
#include <ctime>
using namespace std;
int main () {
	int n;
	int random = 0;
	cout<<"-----RANDOM WHEEL OF FORTUNE-----"<<endl;
	cout<<"Hi, enter the number of elements you want"<<endl;
	cin>>n;
	while(n <= 0) {
		cout<<"Error: number below or equal to 0. Try again"<<endl;
		cin>>n;
	}
	srand(time(0));
	cout<<"What type of elements do you want? (i / I = int, d / D = double, s / S = string, c / C = char)"<<endl;
	char t;
	cin>>t;
	if (t == 'i' || t == 'I') {
		int elements[n];
		for (int i = 0; i < n; i++) {
			cout<<"Enter the element "<<i + 1<<endl;
			cin>>elements[i];
		}
		random = rand() % n;
		cout<<"The random element is "<<elements[random]<<endl;
	} else if (t == 'd' || t == 'D') {
		double elements[n];
		for (int i = 0; i < n; i++) {
			cout<<"Enter the element "<<i + 1<<endl;
			cin>>elements[i];
		}
		random = rand() % n;
		cout<<"The random element is "<<elements[random]<<endl;
	} else if (t == 's' || t == 'S') {
		string elements[n];
		for (int i = 0; i < n; i++) {
			cout<<"Enter the element "<<i + 1<<endl;
			cin>>elements[i];
		}
		random = rand() % n;
		cout<<"The random element is "<<elements[random]<<endl;
	} else if (t == 'c' || t == 'C') {
	    char elements[n];
		for (int i = 0; i < n; i++) {
			cout<<"Enter the element "<<i + 1<<endl;
			cin>>elements[i];
		}
		random = rand() % n;
		cout<<"The random element is "<<elements[random]<<endl;
	}
	return 0;
}